packages/form/src/components/UnitInputField/index.tsx

```diff
@@ -0,0 +1,111 @@
+import { UnitInput } from '@ultraviolet/ui'
+import type { ComponentProps } from 'react'
+import { useController } from 'react-hook-form'
+import type { FieldPath, FieldValues } from 'react-hook-form'
+import { useErrors } from '../../providers'
+import type { BaseFieldProps } from '../../types'
+
+type UnitInputFieldProps<
+  TFieldValues extends FieldValues,
+  TName extends FieldPath<TFieldValues>,
+> = BaseFieldProps<TFieldValues, TName> &
+  Pick<
+    ComponentProps<typeof UnitInput>,
+    | 'id'
+    | 'name'
+    | 'className'
+    | 'data-testid'
+    | 'disabled'
+    | 'value'
+    | 'max'
+    | 'min'
+    | 'options'
+    | 'placeholder'
+    | 'placeholderUnit'
+    | 'size'
+    | 'unitValue'
+    | 'required'
+    | 'width'
+    | 'helper'
+    | 'selectInputWidth'
```

Comment:
I dont like that may be we need another way to handle sizes inside the input ?

What do you think about using a Row and let the user pass a custom templateColumns ? Like this he can manage both side ?